scripts/create-kubestellar-demo-env.sh

```diff
@@ -13,10 +13,38 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Deploys Kubestellar environment for demo purposes.
+# Deploys Kubestellar environment for demo purposes on kind or k3d.
 
 set -e
 
+# Default Kubernetes platform parameter
+k8s_platform="kind"
+
+# Parse command-line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
```

Comment:
As long as we are parsing command line arguments, we should accept `-X` too.

Reply:
added